_sass/_layout.scss

```diff
@@ -243,3 +243,34 @@
 .pagination {
   text-align: center;
 }
+
+// Social Media Biography
+
+.bio-social-media {
+  ul {
```

Comment:
I am happy to do otherwise, but I'm not sure how to implement what you are suggesting. 

I thought that some of these properties had to be targeted to an html element?